src/components/Form.js

```diff
@@ -1,50 +1,38 @@
+/* eslint-disable no-unused-vars */
 import React, { useState } from 'react'
+import './Form.css'
 
-function Form() {
-  const [password, setPassword] = useState('')
-  const [email, setEmail] = useState('')
-
-  const handlePassword = (e) => {
-    setPassword(e.target.value)
-  }
-
-  const handleEmail= (e) => {
-    setEmail(e.target.value)
+export default function Form() {
+  const [value, setValue] = useState('')
+  const [messeges, setMesseges] = useState([])
+  const changeInputMessage = (e) => {
+    setValue(e.target.value)
   }
 
-  const handleSubmit = (e) => {
-    e.preventDefault()
-
-    
-    setPassword('')
-    setEmail('')
+  const sendMessageOnEnter = (e) => {
+    if (e.key === 'Enter') {
+      setMesseges([...messeges, value])
+      setValue('')
+    }
   }
-
   return (
-    <div>
-      <form onSubmit={handleSubmit}>
-        <label>
-          Password
-          <input
-            type="text"
-            value={password}
-            onChange={handlePassword}
-          />
-        </label>
-
-        <label>
-          Email
-          <input
-            type="text"
-            value={email}
-            onChange={handleEmail}
-          />
-        </label>
-
-        <button type="submit">Submit</button>
-      </form>
-    </div>
+    <>
+      <div className="chat">
+        <div className="messege_container">
+          {messeges.map((messege, idx) => (
```

Comment:
Please install this extension

Name: Code Spell Checker
Id: streetsidesoftware.code-spell-checker
Description: Spelling checker for source code
Version: 2.20.5
Publisher: Street Side Software
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker